### ISSUE TITLE
Opens As Notebook for file click

### ISSUE THREAD

**Issue by darwingr** [FelixKratz/JupyterApp-mac#4]
When I double click a file in the Finder it opens it as a notebook but if I use the context menu it opens as a lab.
Is this expected? How can I have it open as a lab from a double click?

**Comment by darwingr** [comment]
ViewController.swift line 106
```
        if (Preferences.shared.useNotebooksOnFolder && file == "" || file != "") {
            app = "notebook"
        }
```
Looks like an intentional yet rational default to me. I might just tweak it directly for now.
I understand that might be a bit more work to add a whole new preference option.

Background:
I'd like to introduce this to some people hitting the limitations of excel and the layouts with Lab tabs a kinda nice. I didn't want to have to explain the differences of context menu vs double click since learning a programming language may already be enough of a hurdle to fit it into their workflow. What you've built here is pretty awesome without making it too complicated and still managing to defer environment configuration to whatever the user defined in the command line. Cheers.

**Comment by FelixKratz** [comment]
You are right in your assumption that this is intentional, I have just coded this up for my specific workflow but I see the appeal of an additional preference for the double click action. I will let you know when I have implement this.

**Comment by FelixKratz** [comment]
I have implemented this as a preference now dc296ccf8b376772711a18630d07c45d3c2de16e

**Comment by darwingr** [comment]
BEAUTY!

Works well dude. Thanks!
